dagobah/core/dagobah.py

```diff
@@ -245,7 +245,8 @@ def add_task_to_job(self, job_or_job_name, task_command, task_name=None,
         job.add_task(task_command, task_name, **kwargs)
         job.commit()
 
-    def add_jobtask_to_job(self, job_or_job_name, target_job, task_name=None):
+    def add_jobtask_to_job(self, job_or_job_name, target_job, task_name=None,
+                           **kwargs):
```

Comment:
The only extra param looks like `hostname`, which should probably not actually be an option for JobTask? If so, the `**kwargs` here and the `hostname` bit in `api.py` can both go.